samples/ASBTaskQueue/Greetings/Adaptors/Data/MigrationSqlHelper.cs

```diff
@@ -12,15 +12,11 @@ public static void ApplyResource(Migration migration, MigrationBuilder migration
             var assembly = Assembly.GetExecutingAssembly();
 
             var name = $"{migration.GetType().Namespace}.{resourceName}";
-            using (var stream = assembly.GetManifestResourceStream(name))
-            {
-                if (stream == null) throw new ArgumentNullException(resourceName);
-                using (var textStreamReader = new StreamReader(stream))
-                {
-                    var sql = textStreamReader.ReadToEnd();
-                    migrationBuilder.Sql(sql);
-                }
-            }
+            using var stream = assembly.GetManifestResourceStream(name);
+            if (stream == null) throw new ArgumentNullException(resourceName);
+            using var textStreamReader = new StreamReader(stream);
```

Comment:
There are nested using here, which can be problematic with using declarations as opposed to using statements because the streamreader and stream may dispose in the wrong sequence

Reply:
done